front/app/containers/Admin/projects/project/analysis/FilterItems/index.tsx

```diff
@@ -77,9 +77,9 @@ const translationKeys: Record<
   },
 };
 
-const FilterItems = ({ filters, isEditable }: FilterItemsProps) => {
+const FilterItems = ({ filters, isEditable, analysisId }: FilterItemsProps) => {
```

Comment:
No, but since we are not always on the analysis screen when we need this component anymore, we need to explicitly pass the analysis id instead of taking it from the route params.